<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeView" name="treeViewLeft">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>390</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>485</y>
      <width>776</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>776</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTreeView" name="treeViewRight">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>390</width>
      <height>400</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="langButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>0</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
